hagrid/settings.py

```diff
@@ -118,6 +120,12 @@
     },
 ]
 
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': [
+        'rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly'
```

Comment:
As we do not need API write permissions, even for logged in users, I'd prefer to explicitly only allow reads from *anyone*.

Reply:
Added custom permission class to only allow same_methods (GET, OPTIONS, HEAD)